### PR TITLE
Popups modification task for VII

### PR DESCRIPTION
**Pull request by asapconet**
Please merge my code this time.
I take God beg you... 
Thank you!

### CODE REVIEW

**Comment by shodown96** [comment]
Woahhh, I thought yours was handled since, Please pull for the last time.
And could you turn your code formatter of, you are creating a bunch of unnecessary additional lines

**Review by shodown96 — CHANGES_REQUESTED** [review]
...

**Comment by asapconet** [comment]
I’ll do that, but my PC is currently inaccessible. And for me formatted ummm, well I’ll try to turn it off too. Thank you.

Sent by ASAP_A1

File: frontend/mainApp/src/utils.js [hunk 1]
```diff
@@ -23,14 +22,17 @@ export const updateProspects = (contacts, id, newDetails) => {
...
-
+export const customAlert = (
+  message,
+  type = "success" || "warning" || "info" || "error"
+) => {
   Swal.fire({
     text: message,
     icon: type,
     showCancelButton: false,
+    timer: 2000,
```
Comment (asapconet) [inline frontend/mainApp/src/utils.js:33]: Just set up a timeout for alert. Cause it was static initaialy.

**Review by asapconet — COMMENTED** [review]
Went through them all again, its all okay from my end.
Basic changes occurred on the stylings.
Especially from them input modal, from prospect.

**Pull request merged**